nectar/src/command/trade.rs

```diff
@@ -69,7 +69,7 @@ pub async fn trade(
     let update_interval = Duration::from_secs(15u64);
 
     let (rate_future, rate_update_receiver) =
-        init_rate_updates(update_interval, settings.maker.kraken_api_host);
+        init_rate_updates(Duration::from_secs(5 * 60), settings.maker.kraken_api_host);
```

Comment:
Maybe it's time to put those durations in `const`s.

Reply:
I don't mind honestly. I believe we are going to be tweaking those a lot so we may even want to put them into the config :)